gradle/libs.versions.toml

```diff
@@ -23,6 +23,7 @@ commons-text = "1.12.0"
 guava = "33.3.1-jre"
 jmustache = "1.15"
 commonmark = "0.23.0"
+spring-boot = "3.3.3"
```

Comment:
don't define a new version from spring boot. Instead [use the version already in `micronaut-spring`](https://github.com/micronaut-projects/micronaut-spring/blob/5.8.x/gradle/libs.versions.toml#L2)

Reply:
I can't, because I need to use some spring modules that are not in the mnSpring versions directory.

Reply:
I have removed it https://github.com/micronaut-projects/micronaut-openapi/pull/1770/commits/453a566bb7a29a7f6cfe0481b19c85946531fc8c I also removed the extra gradle plugin.